src/view/shell/desktop/LeftNav.tsx

```diff
@@ -62,39 +64,119 @@ import {
   UserCircle_Filled_Corner0_Rounded as UserCircleFilled,
   UserCircle_Stroke2_Corner0_Rounded as UserCircle,
 } from '#/components/icons/UserCircle'
+import * as Menu from '#/components/Menu'
+import * as Prompt from '#/components/Prompt'
 import {Text} from '#/components/Typography'
 import {router} from '../../../routes'
 
 const NAV_ICON_WIDTH = 28
 
 function ProfileCard() {
-  const {currentAccount} = useSession()
-  const {isLoading, data: profile} = useProfileQuery({did: currentAccount!.did})
-  const {isDesktop} = useWebMediaQueries()
+  const {currentAccount, accounts} = useSession()
+  const {onPressSwitchAccount, pendingDid} = useAccountSwitcher()
+  const {logoutEveryAccount} = useSessionApi()
+  const {isLoading, data} = useProfilesQuery({
+    handles: accounts.map(a => a.did),
+  })
+  const profiles = data?.profiles
+  const signOutPromptControl = Prompt.usePromptControl()
+  const {gtTablet} = useBreakpoints()
   const {_} = useLingui()
+  const {setShowLoggedOut} = useLoggedOutViewControls()
+  const closeEverything = useCloseAllActiveElements()
+
   const size = 48
 
-  return !isLoading && profile ? (
-    <Link
-      href={makeProfileLink({
-        did: currentAccount!.did,
-        handle: currentAccount!.handle,
-      })}
-      style={[styles.profileCard, !isDesktop && styles.profileCardTablet]}
-      title={_(msg`My Profile`)}
-      asAnchor>
-      <UserAvatar
-        avatar={profile.avatar}
-        size={size}
-        type={profile?.associated?.labeler ? 'labeler' : 'user'}
-      />
-    </Link>
-  ) : (
-    <View style={[styles.profileCard, !isDesktop && styles.profileCardTablet]}>
-      <LoadingPlaceholder
-        width={size}
-        height={size}
-        style={{borderRadius: size}}
+  const onAddAnotherAccount = () => {
+    setShowLoggedOut(true)
+    closeEverything()
+  }
+
+  const profile = profiles?.find(p => p.did === currentAccount!.did)
+  const otherAccounts = accounts
+    .filter(acc => acc.did !== currentAccount!.did)
+    .map(account => ({
+      account,
+      profile: profiles?.find(p => p.did === account.did),
+    }))
+
+  return (
+    <View style={[a.my_md, gtTablet && [a.align_start, a.pl_md]]}>
+      {!isLoading && profile ? (
+        <Menu.Root>
+          <Menu.Trigger label={_(msg`Switch accounts`)}>
+            {({props}) => (
+              <Button label={props.accessibilityLabel} {...props}>
```

Comment:
I think we could use a hover state of some kind here. My first thought would be a little chevron-down to the right, maybe a subtle bg fade in. A no-icon option could be a little border fade-in or something.